checkov/common/sca/output.py

```diff
@@ -311,14 +311,15 @@ def add_to_report_sca_data(
     license_statuses: list[_LicenseStatus],
     sca_details: SCADetails | None = None,
     report_type: str | None = None,
+    dependencies: dict[str, Any] | None = None,
 ) -> None:
     licenses_per_package_map: dict[str, list[str]] = \
         _add_to_report_licenses_statuses(report, check_class, scanned_file_path, rootless_file_path, runner_filter,
                                          license_statuses, sca_details, report_type)
 
     add_to_reports_cves_and_packages(report, check_class, scanned_file_path, rootless_file_path, runner_filter,
                                      vulnerabilities, packages, licenses_per_package_map, sca_details, report_type,
-                                     ScanDataFormat.FROM_TWISTCLI)
+                                     ScanDataFormat.TWISTCLI_FORMAT if dependencies is None else ScanDataFormat.DEPENDENCY_TREE_FORMAT)
```

Comment:
```suggestion
                                     ScanDataFormat.DEPENDENCY_TREE_FORMAT if dependencies else ScanDataFormat. TWISTCLI_FORMAT)
```
then it will also apply, if `dependencies` is an empty `dict`

Reply:
I did it that way because empty `dependencies` means -> we get the results using dependency tree but no dependencies was found 
but if `dependencies` is none that means that we get the results using Twistcli

Reply:
ok, gotcha!